tests/unit/test_miscutils.py

```diff
@@ -33,13 +32,7 @@ def test_versioninfo(tmp_path):
             date=(0, 0)
         ),
         kids=[
-            StringFileInfo(
-                [
-                    StringTable(
-                        '040904b0',
-                        [StringStruct('FileDescription',
-                                      'versioninfo test')])
-                ]),
+            StringFileInfo([StringTable('040904b0', [StringStruct('FileDescription', 'versioninfo test')])]),
```

Comment:
The previous formatting was mimicking the layout of version info. It might make sense to preserve that.